web/packages/core/src/ruffle-player.ts

```diff
@@ -406,6 +407,10 @@ export class RufflePlayer extends HTMLElement {
 
                 if ("url" in options) {
                     console.log("Loading SWF file " + options.url);
+                    this.swfUrl = new URL(
+                        options.url,
+                        document.location.href
+                    ).href;
```

Comment:
URL constructor can fail if you give it an invalid URL, for example, this blows up for me:
`player.load("http://? invalid ?");`
We need to handle this better in general, (cc #1824) but at least we probably shouldn't throw at this point. Could just wrap it in a try/catch for now, defaulting to options.url.

Reply:
Sorry, I should have checked that. I've added the try-catch block now.

I don't think there's much point in adding an error message since whatever solution is proposed for #1282 while likely log the same error.

Let me know if you disagree and I can add a small error message.